style.css

```diff
@@ -687,6 +688,67 @@
   color: #6070ff;
 }
 
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: hidden; /* Enable scroll if needed */
+  background-color: #6070ff;
+  background-color: rgba(96, 112, 255, 0.7); 
+}
+
+
+/* Modal Content */
+.modal-content {
+  background-color: rgba(96, 112, 255, 0.7); /* Fallback color */
+  margin: auto;
+  padding: 20px;
+  width: 100%;
```

Comment:
- [ ] For a correct user interface and a better user experience kindly remove this property to be able to see the close button otherwise the button is outside the mobile container 👇🏽 

| With width: 100%; | Without |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/225009006-0e6523da-a2fa-4c6f-a120-79cbb6e8ce86.png) | ![image](https://user-images.githubusercontent.com/58642949/225009108-65d49b77-a30b-429e-b43a-76e377e8da27.png) |